PRIVACY.md

```diff
@@ -130,45 +149,396 @@ provided by [http://www.aboutads.info/choices/](http://www.youronlinechoices.com
+Tremont Street, 3rd Floor, Boston, MA 02111-1338 („Help Scout“). To process
+your support requests through our contact form or directly to
+support@tideways.io or other support mailboxes, necessary data such as
+Firstname, Lastname, E-Mail address, subject and message are transmitted to
+Help Scout, located in the United States of America. If you contact us via
+phone as part of a customer relationship, we will store your name, phone
+number, date of the call as well as notes and summary of our employees with
+regard to the call and they are transmitted to Help Scout into the United
+States of America.
+
+We only transmit the information above to Help Scout servers into the US. The
+US is a so called insecure third party country. However Help Scout has
+voluntarily committed itself to comply to the US-EU „Privacy Shield“ treaty and
+has committed to implement and uphold EU data privacy regulation.
+
+9.2. Additional information for your privacy can be reviewed in the [Privacy
+Policy](https://www.helpscout.net/company/legal/privacy/) and [Terms of
+Service](https://www.helpscout.net/company/legal/terms-of-service/) of Help
+Scout.
+
+# 13. Website Analysis with Matomo
+
+13.1. For webseite analysis we use Matomo (formerly Piwik), an open-source
+software, which evaluates website visits. Matomo uses cookies to enable the
+analysis of website vists. For this purpose we transmit and store tracking data
+based on the cookie on our servers. IP-addresses are truncated during this
+process such that we cannot directly identify you as a visitor and you stay
+anonym. The resulting pseudonymized profiles are used soley to optimize our
+webseite content and usability.
+
+13.2. You can prevent the use of cookies on our website through settings in
+your browser, but it may be possible that you cannot use all features of our
+website to their full extend.
+
+13.3. If you don‘t want to allow us to the storing and analysis of your visitor
+data, then you can object to our use of Matomo at any time. In this case your
+browser will store a so called opt-out cookie that prevents Matomo from storing
+any data on our servers.
+
+13.4. To object our use of Cookies for website analysis click here.
+
+# 14. Google Gsuite
+
+14.1. For our e-mail correspondance and for storing files and documents we use
+Gmail, Drive and Gsuite, a service by Google Inc. (www.google.de), 1600
+Amphitheatre Parkway, Mountain View, CA 94043, USA („Google“).
+
+14.2. Google processes your data in relation with e-mail correspondences and
+storage of documents such as contracts, invoices and as well data to optimize
+and analyse Gsuite. For this we have closed a model contract clause with
+Google, which obligates Google to proect your data, to  process it with regard
+to this privacy policy and not to pass it on to third parties.
+
+14.3. Additional information to privacy can be reviewed in Google‘s [privacy
+policy](https://policies.google.com/privacy).
+
+# 15. Erasing and Blocking of Data
```

Comment:
The following paragraphs §15 - §30 clarify again all the GDPR granted rights you have towards us and that we will comply with them as requested by the regulation. Take note that we grant these rights to all users of Tideways, independent of their European Citizenship.